scene/2d/skeleton_2d.cpp

```diff
@@ -787,7 +787,7 @@ void Skeleton2D::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("set_modification_stack", "modification_stack"), &Skeleton2D::set_modification_stack);
 	ClassDB::bind_method(D_METHOD("get_modification_stack"), &Skeleton2D::get_modification_stack);
-	ClassDB::bind_method(D_METHOD("execute_modifications", "execution_mode", "execution_mode"), &Skeleton2D::execute_modifications);
+	ClassDB::bind_method(D_METHOD("execute_modifications", "p_delta", "execution_mode"), &Skeleton2D::execute_modifications);
```

Comment:
Should be `delta`. `p_` prefix is for internal C++ code, not bindings.

Reply:
ugh, haste makes waste :)